examples/tools/scene_viewer.rs

```diff
@@ -0,0 +1,407 @@
+use bevy::{
```

Comment:
The "select a model" instructions should be mirrored at the top here, as a `//!` doc comment.